docs/apache-airflow-providers-google/operators/cloud/gcs.rst

```diff
@@ -121,6 +121,19 @@ More information
 See Google Cloud Storage insert documentation to `create a ACL entry for ObjectAccess
 <https://cloud.google.com/storage/docs/json_api/v1/objectAccessControls/insert>`_.
 
+.. _howto/operator:GCSListObjectsOperator:
+
+Listing Objects in a Bucket
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Listing Objects in a Bucket allows you to view contents of bucket in the Google Cloud Storage.
+
+It is performed through the
+:class:`~airflow.providers.google.cloud.operators.gcs.GCSListObjectsOperator` operator.
+
+.. seealso::
+    For more information on how to use this operator, take a look at the guide:
+    :ref:`howto/operator:GCSListObjectsOperator`
```

Comment:
Is this just linking to this same section? Have you tried [building the documentation locally](https://github.com/apache/airflow/blob/main/docs/README.rst#building-documentation)?

Reply:
PS would be good to add a link to the document that I shared in the [contributing docs](https://github.com/apache/airflow/tree/main/contributing-docs)

Reply:
Agreed - Am thinking of adding https://github.com/apache/airflow/blob/main/contributing-docs/10_working_with_git.rst especially on the rebase line, so users can follow along from there.